src/main/java/net/dv8tion/jda/core/entities/Message.java

```diff
@@ -63,7 +63,16 @@
  */
 public interface Message extends ISnowflake, Formattable
 {
-    int MAX_FILE_SIZE = 8 << 20; // 8mb
+    /**
+     * Useful constant for the max allowed file size for message attachments on normal accounts. Value: {@value #MAX_FILE_SIZE} Bytes (8 MiB)
+     */
+    int MAX_FILE_SIZE = 8 << 20; // 8 MiB
+
+    /**
+     * Useful constant for the max allowed file size for message attachments on nitro accounts. Value: {@value #MAX_FILE_SIZE_NITRO} Bytes (50 MiB)
+     */
+    int MAX_FILE_SIZE_NITRO = 50 << 20; // 50 MiB
```

Comment:
This isn't really consistent you are calling your Boolean premium to check if user is a Nitro users but here you are calling you constant MAX_FILE_SIZE_NITRO instead of MAX_FILE_SIZE_PREMIUM. I'd say either rename the Boolean to nitro or rename this constant.
Your comment is "//50 MiB" is wrong the current int value would be 50mb.